natlas-server/app/static/util/system-status.ts

```diff
@@ -7,13 +7,21 @@ export function updateStatus(): void {
 	})
 	.then((data) => {
 		var progbar = document.getElementById("cycle_status")
+		var progbg = document.getElementById("cycle_status_overlay")
 		var width = (data['scans_this_cycle'] / data['effective_scope_size']) * 100;
 		progbar.setAttribute('aria-valuenow', width.toString());
 		progbar.setAttribute('style', `width:${width}%;`);
 		const entries = Object.entries(data);
 		for (const [key, val] of entries) {
 			document.getElementById(key).innerText = val.toString();
 		}
-		document.getElementById('prog_percent').innerText = Math.round(width)+"%";
+		progbg.innerText = Math.round(width)+"%";
```

Comment:
I have no idea why CodeClimate isn't complaining about this code. We're missing semicolons, but it's flagged as an error, we're using var instead of const, etc. Do you know why it's not flagging this code? Are we missing something for typescript?

Reply:
Added tslint and configured a few rules to match preference.